### PR TITLE
PMD Problems - FixationPoint and FixationSequence

### PR DESCRIPTION
**Pull request by Tohaker**
### PMD Problems
`FixationPoint` 
[FormalParameterNamingConventions](https://pmd.github.io/pmd-6.17.0/pmd_rules_java_codestyle.html#formalparameternamingconventions) - line 28

`FixationSequence` 
[AvoidReassigningParameters](https://pmd.github.io/pmd-6.17.0/pmd_rules_java_bestpractices.html#avoidreassigningparameters) - line 34
[LocalVariableNamingConventions](https://pmd.github.io/pmd-6.17.0/pmd_rules_java_codestyle.html#localvariablenamingconventions) - line 63

### Solution
Replaced the `FixationPoint` POJO implementation with `lombok` annotations for simplicity. Results in minor change of `toString()` implementation, but not substantial.

Refactored some of `FixationSequence` for better readability.

### Extras
* 98% Test coverage

### CODE REVIEW

File: gazeplay-commons/src/test/resources/GazePlay.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #Test File
-#Tue Feb 25 19:51:54 GMT 2020
+#Fri Feb 28 22:19:15 GMT 2020
```
Comment (YannRobert) [inline gazeplay-commons/src/test/resources/GazePlay.properties]: ?

File: gazeplay-commons/src/main/java/net/gazeplay/commons/utils/FixationPoint.java [hunk 2]
```diff
@@ -1,40 +1,14 @@
...
-import lombok.Setter;
+import lombok.*;
 
+@Data
+@AllArgsConstructor
 public class FixationPoint {
 
-    @Setter
-    @Getter
     private long timeGaze;
```
Comment (YannRobert) [inline gazeplay-commons/src/main/java/net/gazeplay/commons/utils/FixationPoint.java]: final

File: gazeplay-commons/src/main/java/net/gazeplay/commons/utils/FixationPoint.java [hunk 3]
```diff
@@ -1,40 +1,14 @@
...
-    @Setter
-    @Getter
     private long gazeDuration;
```
Comment (YannRobert) [inline gazeplay-commons/src/main/java/net/gazeplay/commons/utils/FixationPoint.java]: final

File: gazeplay-commons/src/main/java/net/gazeplay/commons/utils/FixationPoint.java [hunk 4]
```diff
@@ -1,40 +1,14 @@
...
-    @Setter
-    @Getter
     private int x;
```
Comment (YannRobert) [inline gazeplay-commons/src/main/java/net/gazeplay/commons/utils/FixationPoint.java]: final

File: gazeplay-commons/src/main/java/net/gazeplay/commons/utils/FixationPoint.java [hunk 5]
```diff
@@ -1,40 +1,14 @@
...
-    @Setter
-    @Getter
     private int y;
```
Comment (YannRobert) [inline gazeplay-commons/src/main/java/net/gazeplay/commons/utils/FixationPoint.java]: final

File: gazeplay-commons/src/test/resources/GazePlay.properties (hunk 1 above)
Comment (Tohaker) [reply gazeplay-commons/src/test/resources/GazePlay.properties]: I'm sure I prevented this from incrementing, its handled automatically. I'll see if I can fix it

File: gazeplay-commons/src/main/java/net/gazeplay/commons/utils/FixationPoint.java (hunk 2 above)
Comment (Tohaker) [reply gazeplay-commons/src/main/java/net/gazeplay/commons/utils/FixationPoint.java]: Why should these be final? They had setters for a reason before and this doesn't remove those setters.

**Comment by schwabdidier** [comment]
@Tohaker conflit

**Pull request merged**